hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManager.java

```diff
@@ -186,10 +186,11 @@ OmKeyLocationInfo allocateBlock(OmKeyArgs args, long clientID,
    * the key name and all its associated block IDs. A pending open key has
    * prefix #open# in OM DB.
```

Comment:
Is this line in the original documentation still relevant? It looks like open keys are placed in the open key table unprefixed.